Control coverage/Feature/SQLServer.md

```diff
@@ -0,0 +1,11 @@
+| <b>ControlId:</b><br>Azure_SQLDatabase_AuthZ_Firewall_Deny_Access_AzureServices<br><b>DisplayName:</b><br>Use the 'Allow access to Azure services' flag only if required<br><b>Description: </b><br> Use the 'Allow access to Azure services' flag only if required |<b>ARM API to get Firewall rules for SQLServer: </b> <br> /subscriptions/{subscriptionId}<br>/resourceGroups/{resourceGroupName}<br>/providers/Microsoft.Sql<br>/servers/{serverName}<br>/firewallRules <br> ?api-version=2014-04-01 <br><b>Properties:</b><br> properties.FirewallRuleList | <b>Passed: </b><br>Allow access to Azure services flag is on.<br><b>Failed: </b><br>Allow access to Azure services flag is off. |
+| <b>ControlId:</b><br>Azure_SQLDatabase_AuthZ_Use_AAD_Admin<br><b>DisplayName:</b><br>Use AAD Authentication for SQL Database<br><b>Description: </b><br> Enable Azure AD admin for the SQL Database |<b>ARM API to get Administrators of SQLServer: </b> <br> /subscriptions/{subscriptionId}<br>/resourceGroups/{resourceGroupName}<br>/providers/Microsoft.Sql<br>/servers/{serverName}<br>/administrators<br>?api-version=2014-04-01 <br><b>Properties:</b><br> properties.IsADAdministratorEnabled | <b>Passed: </b><br>Active Directory admins are assigned on SQL Server.<br><b>Failed: </b><br>No Active Directory admin is assigned on SQL Server. |
+| <b>ControlId:</b><br>Azure_SQLDatabase_DP_Enable_TDE<br><b>DisplayName:</b><br>Transparent data encryption (TDE) must be enabled<br><b>Description: </b><br> Transparent data encryption (TDE) must be enabled |<b>ARM API to get Databases of SQLServer: </b> <br> /subscriptions/{subscriptionId}<br>/resourceGroups/{resourceGroupName}<br>/providers/Microsoft.Sql<br>/servers<br>?api-version=2019-06-01-preview <br><b>Properties:</b><br> properties.Databases <br> Databases.TDEStatus | <b>Passed: </b><br>All databases for SQL server have transparent data encryptoin enabled, or <br>No database is present for SQL server.<br><b>Failed: </b><br>At least one of the databases for SQL server do not have transparent data encryption enabled. |
```

Comment:
Typo - `encryptoin` to be changed to `encryption`.

Reply:
Done.